### ISSUE TITLE
[BUG] A focus highlight was found to fail non-text contrast minimum requirements.

### ISSUE THREAD

**Issue by KatiePUX** [UKHO/admiralty-design-system#180]
Non-Text Contrast

A focus highlight was found to fail non-text contrast minimum requirements.

Outcome of DAC Accessibility Audit:
[https://ukho.sharepoint.com/sites/EXT_Data_Upload/Shared%20Documents/Forms/AllItems.aspx?OR=Teams%2DHL&CT=1711376954679&clickparams=eyJBcHBOYW1lIjoiVGVhbXMtRGVza3RvcCIsIkFwcFZlcnNpb24iOiIyNy8yNDAyMjkyMDYwMiIsIkhhc0ZlZGVyYXRlZFVzZXIiOmZhbHNlfQ%3D%3D&id=%2Fsites%2FEXT%5FData%5FUpload%2FShared%20Documents%2FGeneral%2FAccessibility%20Audit%2FPrivate%20Beta%2FDAC%20Accessibility%20Report%20WCAG%202%2E2%20UKHO%20UDU%20130324%20Final%20Version%2Epdf&viewid=8b1f0a9e%2D2897%2D4a4d%2D8283%2Dcc660e97e3e4&parent=%2Fsites%2FEXT%5FData%5FUpload%2FShared%20Documents%2FGeneral%2FAccessibility%20Audit%2FPrivate%20Beta](url)

(Page 39 - 40)

WCAG Reference:
**1.4.11 Non-text Contrast (Level AA)**
Understanding Non-text Contrast | How to Meet Non-text Contrast
Issue ID: DAC_Non-Text_Contrast_01
URL: https://ukhodataupload.admiralty.co.uk/external/dashboard
Page title: Dashboard
Journey: 1.2

The highlight displayed when focusing elements on the page fails to meet the minimum
contrast ratio requirements. This instance can be found when a yellow focus indicator is 
provided on a white background. Due to the colours used for the indicator and the 
background, users with a visual impairment that affects the perception of colour may find it 
difficult to distinguish between focused and unfocused elements, which can make it difficult 
to navigate the service.

**Comment by buzzub** [comment]
In addition to the focus highlight in the header not being accessible, other components have potential accessibility issues and also need resolving:

- breadcrumb
- button
- checkbox
- colour-block
- expansion
- file-input
- header
- paginator
- radio
- read-more
- side-nav
- tabs
